Formula/unrtf.rb

```diff
@@ -21,7 +21,13 @@ class Unrtf < Formula
 
   def install
     system "./bootstrap"
-    system "./configure", "LIBS=-liconv", "--prefix=#{prefix}"
+    args = %W[
+      --prefix=#{prefix}
+    ]
```

Comment:
```suggestion
    args = %W[--prefix=#{prefix}]
```
We don't need this much whitespace here. (Sorry I didn't notice earlier)